templates/components/itilobject/timeline/form_validation.html.twig

```diff
@@ -286,5 +300,70 @@
             window.scrollTo(0,document.body.scrollHeight);
          </script>
       {% endif %}
+
+      <script type="text/javascript">
+         function waitForElement{{ rand }}(selector) {
+            return new Promise(resolve => {
+               if (document.querySelector(selector)) {
+                     return resolve(document.querySelector(selector));
+               }
+
+               const observer = new MutationObserver(mutations => {
+                     if (document.querySelector(selector)) {
+                        resolve(document.querySelector(selector));
+                        observer.disconnect();
+                     }
+               });
+
+               observer.observe(document.body, {
+                     childList: true,
+                     subtree: true
+               });
+            });
+         }
```

Comment:
Maybe `waitForElement` could be added to `common.js`. It looks like a generic function.